capa/features/extractors/viv/insn.py

```diff
@@ -488,7 +488,7 @@ def extract_insn_segment_access_features(f, bb, insn):
 
 def get_section(vw, va):
     for start, length, _, __ in vw.getMemoryMaps():
-        if va and start <= va < start + length:
+        if start <= va < start + length:
```

Comment:
As #442 has been merge, we can remove this check that this code hide some other bug in the future.